### DISCUSSION TITLE
Is there a way to back up mod config?

### DISCUSSION THREAD

**Discussion by Arafor** [xivdev/Penumbra#289] [Q&A] [answered]
So last time I had to reinstall my PC after a huge PC part upgrade. While I was able to back up the mods folder that penumbra used. I was, however, not able to back up to config for all the mods. So after installing FFXIV and getting penumbra again. Adding all the mods back was easy. Organizing them, enabling them, setting priorities and going through all the individual mod options took a lot of time.
Is there a way to do a back up for all the config?

**Comment by Theo-Asterio** [comment, marked as answer]
Yes, a way already exists. There's rolling backups stored at: %appdata%\XIVLauncher\backups\Penumbra 
Otherwise there is also the following picture with the info.
![Penumbra-2nd-PC](https://user-images.githubusercontent.com/58579310/223078538-7ef52930-60e6-47c4-a85b-d5b040e93b31.png)

**Comment by Arafor** [comment]
@Theo-Asterio Thank you so much! Somehow managed to miss this while digging around myself

**Comment by Blackstar90210** [comment]
I found this thread precisely because of my research on Penumbra backups. The problem, however, I was unable to solve. The files found within the path "C:\Users\'name'\AppData\Roaming\Roaming\XIVLauncher\backups\Penumbra" seem not to work at all. How come? Have I misunderstood something?

**Reply by Theo-Asterio** [reply]
Because that's the wrong location you're saying, you need to copy in %appdata%\XIVLauncher\backups\Penumbra as literal as you can into the address bar, not the search bar. See the image below for how to do it. You also need to take the zip folder from there, and extract it into %appdata%\XIVLauncher\pluginConfigs and overwrite the old files while the plugin is either disabled or the game is not started.
![How_to_find_XIVLauncher_Folder](https://github.com/xivdev/Penumbra/assets/58579310/d64001ea-0c05-438d-b576-82bfeaaac2ad)

**Reply by Blackstar90210** [reply]
Thank you so much!
I did not see the second position. I'm very sorry

**Comment by guardianofroin** [comment]
I am currently trying to move from Windows to Linux. Is there a way to do this to the Linux setup? I figure the location is going to be different due to the different file structure.

**Reply by thecasualtrex** [reply]
Not sure if you still need the file structure for Linux but you should have a `.xlcore` folder in the home directory.  `~/.xlcore/pluginConfigs` is what you are looking for. The Mods folder you specify yourself when you setup Penumbra. 

In my case as well inside the .xlcore folder there was also a folder called 'backups' that has some plugin backups as well.
